libethereum/State.cpp

```diff
@@ -541,6 +541,7 @@ std::pair<ExecutionResult, TransactionReceipt> State::execute(EnvInfo const& _en
 	u256 const startGasUsed = _envInfo.gasUsed();
 	executeTransaction(e, _t, onOp);
 
+	res.changeLog = m_changeLog;
```

Comment:
I don'like that this way we will have this in `ExecutionResult` (paying with additional memory consumption) even when we don't need it (we need it only for tests and maybe vmtraces)

@chfast what do you think?